.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
+go:
+  - 1.10.2
+
+services:
+  - docker
+sudo: required
```

Comment:
have you tried `false` here? I know when true travis spins up a whole VM vs. a container if false. I know it's needed for some stuff so I'd be curious what the need is

Reply:
Looks like we can go containers, https://docs.travis-ci.com/user/reference/overview/#Virtualization-environments

Reply:
good point I don't know if it is needed at all or not. I'll try without.

Reply:
Sweet, `sudo: false` should be the only change here and you'll notice a huge increase in build time